src/components/HomePage/styles.scss

```diff
@@ -2,4 +2,6 @@
   display: flex;
   flex-direction: row;
   align-items: stretch;
+  height: 100%;
+  background-color: $primaryBlue;
```

Comment:
jw whats this

Reply:
me exploring for real to see what was what
i'll remove